sky/cli.py

```diff
@@ -3336,6 +3348,43 @@ def spot_queue(all: bool, refresh: bool, skip_finished: bool):
 _add_command_alias_to_group(spot, spot_queue, 'status', hidden=True)
 
 
+@spot.command('cost', cls=_DocumentedCodeCommand)
+@click.option(
+    '--refresh',
+    '-r',
+    default=False,
+    is_flag=True,
+    required=False,
+    help='Query the latest statuses, restarting the spot controller if stopped.'
+)
+@click.option('--condensed',
+              '-c',
+              default=False,
+              is_flag=True,
+              required=False,
+              help='If true, do not show verbose breakdown of spot job costs.')
+@usage_lib.entrypoint
+# pylint: disable=redefined-builtin
+def spot_cost_report(refresh: bool, condensed: bool):
+    """Show cost report of managed spot jobs.
+    """
+    click.secho('Fetching managed spot job costs...', fg='yellow')
+    no_costs_found_str = '  No job costs found.'
+    verbose = not condensed
+    try:
+        cost_table = core.spot_cost_report(refresh, verbose)
+    except exceptions.ClusterNotUpError:
+
+        return
```

Comment:
Would be good to check the `cli._get_spot_jobs` to show the outputs when the `cost_table` is not available for better UX.